src/util/embed.py

```diff
@@ -0,0 +1,52 @@
+"""Copyright by CookieDecide, Darkuuu
+Licensed under MIT License
+Contains various helper functions to create embeds for sending. 
+Assures consistent look over different modules.
+"""
+
+import discord
+from discord.ext import commands
+
+
+def create_embed_image(title, description, img_url, url) -> discord.Embed:
+    """Creates an embed showing an image.
+
+    Args:
+        title: Title of the embed.
+        description: Description/Message of embed.
+        img_url: URL pointing to the image.
+        url: URL pointing to the webpage the image is embedded on.
+             If not existing, provide img_url here.
+
+    Returns:
+        An embed with blue color showing an image.
+    """
+    color = 0x14D8FA
```

Comment:
Maybe we can agree to add a comment with the approximate color, like "blue", for better readability.

Reply:
Again, will call you later regarding design decisions in code. Maybe adding a seperate module to specify different colour schemes might be an idea